src/context/Store.js

```diff
@@ -43,7 +44,8 @@ const Store = ({children}) => {
     users: React.useReducer(usersReducer, usersState),
     domains: React.useReducer(domainReducer, domainState),
     initialStartup: React.useReducer(initialStartupReducer, initialStartupState),
-    comments: React.useReducer(commentReducer, commentState)
+    comments: React.useReducer(commentReducer, commentState),
+    chat: React.useReducer(chatReducer, chatState)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `chat` reducer and state to the context provider is a good step towards managing chat-related states globally. However, it's important to ensure that the `chatReducer` handles all possible actions correctly and that the initial `chatState` is defined appropriately.
```diff
     comments: React.useReducer(commentReducer, commentState),
+    chat: React.useReducer(chatReducer, chatState)
   };
   return <Context.Provider value={rootReducer}>{children}</Context.Provider>;
 };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->